NEWS.md

```diff
@@ -1,46 +1,2 @@
-Bio.jl v0.5.0 Release Notes
+Bio.jl v0.6.0 Release Notes
```

Comment:
Shouldn't this be v0.5.0 as it refers to Bio.jl not Julia?

Reply:
Oh yes, that was accidental!